docs/admininstrator-guide/administrator-slash-commands.md

```diff
@@ -1,2 +1,27 @@
 # Administrator Slash Commands
```

Comment:
@jfrerich, I noted that the following four commands are listed in the code but not in the help text:
executeDebugStatsReset
executeDebugStatsSave
executeWebhookURL
executeInstanceList

Should they be added here?

Reply:
I don't think so....

Reply:
@aaronrothschild, I feel like it would be helpful to add the `/jira instance list` command to the help text, but leave out the others.